pages/index.js

```diff
@@ -1,7 +1,8 @@
-import Head from "next/head";
-import styles from "../styles/Home.module.css";
-import Link from "next/link";
-import sanityClient from "../lib/client";
+import Head from 'next/head'
```

Comment:
imports should be in the following order:

- imports from node_modules
- imports of local components
- import(s) of style